cmd/memberagent/main.go

```diff
@@ -192,6 +198,18 @@ func buildHubConfig(hubURL string, useCertificateAuth bool, tlsClientInsecure bo
 			hubConfig.TLSClientConfig.CAData = caData
 		}
 	}
+
+	if header, ok := os.LookupEnv("HUB_KUBE_HEADER"); ok {
+		r := textproto.NewReader(bufio.NewReader(strings.NewReader(header)))
+		h, err := r.ReadMIMEHeader()
+		if err != nil && errors.Is(err, io.EOF) {
```

Comment:
Hi Mitch! Just in case other types of errors happen here, would it be better if we just check against `err`?